### PR TITLE
Brownfield docs

### PR DESCRIPTION
**Pull request by owen-lynch-ibm**
Minor changes for documentation to add references to Adopt in the openstack user-guide Resources chapter 
Issue: `tnc-o-tracking/issues/86`

### CODE REVIEW

**Comment by owen-lynch-ibm** [comment]
Changed base from `master` to `brownfield`

File: docs/user-guide/resources.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # Resources
 
-The Openstack driver allows you to create Stacks in a target Openstack as part of a `Create` lifecycle transition (then remove the Stack with `Delete`). This is done by configuring the use of the driver on Create/Delete and by including Heat or Tosca templates in your Resource package. 
+The Openstack driver allows you to create new or adopt pre-existing Stacks in a target Openstack as part of a `Create` or `Adopt` lifecycle transition (then remove the Stack with `Delete`). This is done by configuring the use of the driver on Create/Adopt/Delete and by including Heat or Tosca templates in your Resource package.
```
Comment (eflamini) [inline docs/user-guide/resources.md:3]: Tosca templates are not supported for Adopt. Are they? If so, then I would make that clear.

Comment (sglover) [reply docs/user-guide/resources.md:3]: The Openstack driver does support TOSCA templates.

**Review by sglover — APPROVED** [review]

File: docs/user-guide/resources.md (hunk 1 above)
Comment (owen-lynch-ibm) [reply docs/user-guide/resources.md]: I clarified that tosca/heat file only supplied in the case of `create`

**Review by sglover — COMMENTED** [review]
Just thought, we should probably document the default Adopt stack states and how to configure them.

**Comment by owen-lynch-ibm** [comment]
> Just thought, we should probably document the default Adopt stack states and how to configure them.
Added a section called 'Adoptable Stack States' :

https://github.com/IBM/openstack-vim-driver/blob/brownfield-docs/docs/user-guide/resources.md#adoptable-stack-states

**Review by sglover — APPROVED** [review]

**Pull request merged**